clang/unittests/Format/FormatTest.cpp

```diff
@@ -26417,6 +26417,20 @@ TEST_F(FormatTest, BreakAfterAttributes) {
                "void g() {}",
                CtorDtorCode, Style);
 
+  Style.ReferenceAlignment = FormatStyle::ReferenceAlignmentStyle::RAS_Left;
+  constexpr StringRef OperatorOverloadCode(
```

Comment:
Don't put the code into a variable, you are only using it once.

Reply:
Fixed